samples/README.md

```diff
@@ -33,3 +33,25 @@ Useful parameter values:
 ## Components source
 
 All samples use pre-built components. The command to run for each container is built into the pipeline file.
+
+## (Optional) Add sample test coverage
```

Comment:
Some of the following are conventions required by the sample tests and some are steps to take in order to add the sample test. Would you mind separating these two. For example, ''sample conventions" and "add sample test"

Reply:
Done.